src/impl/EspNowNode.cpp

```diff
@@ -23,6 +22,9 @@
 // between each message.
 #define NUMBER_OF_RETRIES_FOR_CHALLENGE_REQUEST 50
 
+// wifi channels that will be tried during discovery
+#define WIFI_SCAN_CHANNELS {1,2,3,4,5,6,7,8,9,10,11,12,13,14}
```

Comment:
This doesn't look like it have been formatted with clang-format. Could it be so that you don't have clang-format enabled/installed?

Reply:
You're right I don't have that installed. I'll install it.

Reply:
fixed